src/module/item/sheet/base.ts

```diff
@@ -92,7 +92,7 @@ export class ItemSheetPF2e<TItem extends ItemPF2e> extends ItemSheet<TItem> {
             rollData,
             async: true,
         });
-        enrichedContent.gmNotes = await TextEditor.enrichHTML(item.system.description.gm.trim(), {
+        enrichedContent.gmNotes = await TextEditor.enrichHTML(item.system.description.gm ? item.system.description.gm.trim() : "", {
```

Comment:
```suggestion
        enrichedContent.gmNotes = await TextEditor.enrichHTML(item.system.description.gm, {
```
Linting would fail for this line because it is too long. Also I don't see a reason for that `trim` to be there in the first place so removing it should be a better fix.